paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -456,6 +456,8 @@ internal class PaymentSheetViewModel @Inject internal constructor(
                     label = args.googlePayConfig?.label,
                 )
             }
+        } else if (paymentSelection is PaymentSelection.ExternalPaymentMethod) {
+            ExternalPaymentMethodHandler.confirm(_paymentSheetResult::tryEmit)
```

Comment:
I _think_ we can wait to test the confirm cases until it's truly implemented.

Reply:
sgtm, I think we will be able to write better tests once it's actually implemented